data/sql/updates/pending_db_world/rev_1609681055620795500.sql

```diff
@@ -0,0 +1,116 @@
+UPDATE `creature_template` SET `mindmg` = 27486, `maxdmg` = 36882, `DamageModifier` = 1.01 WHERE `entry` = 29446;
+UPDATE `creature_template` SET `mindmg` = 13743, `maxdmg` = 18441, `DamageModifier` = 1.01 WHERE `entry` = 15930;
+UPDATE `creature_template` SET `mindmg` = 27486, `maxdmg` = 36882, `DamageModifier` = 1.01 WHERE `entry` = 29447;
+
+/* KEL'THUZAD SUMMONS */
+UPDATE `creature_template` SET `mindmg` = 3120, `maxdmg` = 4360, `DamageModifier` = 1.03 WHERE `entry` = 16427;
+UPDATE `creature_template` SET `mindmg` = 6240, `maxdmg` = 8720, `DamageModifier` = 1.03 WHERE `entry` = 30015;
+UPDATE `creature_template` SET `mindmg` = 3480, `maxdmg` = 4550, `DamageModifier` = 1.03 WHERE `entry` = 16428;
+UPDATE `creature_template` SET `mindmg` = 6960, `maxdmg` = 9060, `DamageModifier` = 1.03 WHERE `entry` = 30048;
+UPDATE `creature_template` SET `mindmg` = 2630, `maxdmg` = 3800, `DamageModifier` = 1.03 WHERE `entry` = 16429;
+UPDATE `creature_template` SET `mindmg` = 5260, `maxdmg` = 7600, `DamageModifier` = 1.03 WHERE `entry` = 30018;
+UPDATE `creature_template` SET `mindmg` = 3480, `maxdmg` = 4550, `DamageModifier` = 1.03 WHERE `entry` = 16441;
+UPDATE `creature_template` SET `mindmg` = 6960, `maxdmg` = 9060, `DamageModifier` = 1.03 WHERE `entry` = 30057;
+
+/* THE ARACHNID QUARTER */ 
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 15956;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29249;
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 15953;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29268;
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 15952;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29278;
+
+/* THE PLAGUE QUARTER */
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 15954;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29615;
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 15936;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29701;
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 16011;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29718;
+
+/* THE MILITARY QUARTER */
+UPDATE `creature_template` SET `mindmg` = 17815, `maxdmg` = 23905, `DamageModifier` = 1.01 WHERE `entry` = 16061;
+UPDATE `creature_template` SET `mindmg` = 35630, `maxdmg` = 47810, `DamageModifier` = 1.01 WHERE `entry` = 29940;
```

Comment:
Nice joke, now where is the damage?

let's take example of this clip...

Instructor razuvious (10-mode) **HITS** a tank of 3.7k GS who has active "Shield wall" which reduces damage taken by 60%, with a basic melee attack... and does **10,256 physical damage.**

where is the total?

let's calculate that...

total MINUS 60% of Shield Wall = damage to character

damage to character MINUS 60% physical damage reduce by flat armor = damage taken

damage taken = 10,256

![image](https://user-images.githubusercontent.com/61223313/103576555-3b96d380-4ecb-11eb-875b-f42c6cd2a753.png)

![image](https://user-images.githubusercontent.com/61223313/103576618-536e5780-4ecb-11eb-8e50-7cf8eabfa506.png)

Reply:
I'm working on it, told ya to not worry about it.

We doing full checks, cause some of them will need even lower due mechanics.

Reply:
"We doing full checks, cause some of them will need even lower due mechanics."

Hope they will come with full and detailed explanation on those cases ..   : )